tests/main/hidden-snap-dir/task.yaml

```diff
@@ -1,5 +1,7 @@
 summary: Check that the experimental hidden dir feature migrates the dir
 
+systems: [-centos-*]
```

Comment:
Please add a small comment why this is disabled here (it's fine to say it fails and reason unknown :)